presto-hive-hadoop2/bin/run_hive_tests.sh

```diff
@@ -23,7 +23,6 @@ set +e
   -Dhive.hadoop2.metastoreHost=localhost \
   -Dhive.hadoop2.metastorePort=9083 \
   -Dhive.hadoop2.databaseName=default \
-  -Dhive.hadoop2.metastoreHost=hadoop-master \
```

Comment:
I think the first one (localhost) was redundant, and second one was actually used.

if i am correct and  your intention is to change hostname from h-m to localhsot, please reflect that in cmt msg

Reply:
btw does it mean we can get rid of https://github.com/prestosql/presto/blob/b8729fe4a117a65a273ca259844f9289e8d1700f/presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHive.java#L48 now?

Reply:
This makes it consistent with `run_hive_s3_tests.sh`.

I will change commit name